README.md

```diff
@@ -135,7 +135,7 @@ $ cat <<EOF | kubectl apply -f -
 apiVersion: pkg.crossplane.io/v1
 kind: Provider
 metadata:
-name: crossplane-provider-oci
+name: provider-oci-family
```

Comment:
From my understanding you don't really install the family correct? You would typically install a sub package like `networking` then the family would get installed. I know there is the issue with the subpackage + family install right now, but do we want to tell users to install the family only? Will they be able to do anything with just the family?